src/event/handlers/change_events.js

```diff
@@ -19,6 +19,10 @@ var changeDoesNotBubble = (function() {
   // @license MIT
   var d = document.createElement('div');
   d.setAttribute('onchange', 't');
+  if (typeof d.attributes === 'undefined') {
+    // Element.attributes is undefined in an node environment
+    return;
```

Comment:
We could use `getAttribute` instead of accessing the attributes object directly I think we can remove the conditional.

See:

https://github.com/Raynos/min-document/blob/master/dom-element.js#L139-L141